pyproject.toml

```diff
@@ -29,12 +29,20 @@ packages = [{ include = "strawberry_django_plus" }]
 [tool.poetry.dependencies]
 python = "^3.8"
 django = ">= 3.2"
-strawberry-graphql = ">= 0.115.0"
+strawberry-graphql = "^0.116.2"
 strawberry-graphql-django = ">= 0.3"
 typing-extensions = ">= 4.1.0"
 django-debug-toolbar = { version = ">=3.4", optional = true }
 django-choices-field = { version = ">=2.0", optional = true }
 
+# docs-deps
+mkdocs = { version = "^1.3.0", optional = true}
+mkdocs-markdownextradata-plugin = {version = "^0.2.5", optional = true}
+mkdocs-material = {version = "^8.3.8", optional = true}
+mkdocs-minify-plugin =  {version = "^0.5.0", optional = true}
+pymdown-extensions = {version = "^9.5", optional = true}
+markdown = {version = "^3.3.7", optional = true}
```

Comment:
Those dependencies should be on `dev-dependencies`, and there's no need to make them optional there. dev dependencies are fine to be installed all the time